Jenkinsfile_parameterized

```diff
@@ -14,5 +14,7 @@ properties([
 ])
 
 withParameterizedPipeline(params.TYPE, params.PRODUCT_NAME, params.APP, params.ENVIRONMENT, params.SUBSCRIPTION) {
-
+  after('functionalTest:${environment}') {
+    archiveArtifacts artifacts: 'REPORTS/**/*.*'
```

Comment:
Allowing empty archive true will also help if for any reason it does not create reports
archiveArtifacts allowEmptyArchive: true, artifacts: 'reports/**'